package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "homework-01",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/app.js",
+  "scripts": {
+    "build": "npx babel src --out-dir dist",
+    "start": "npm run clean && npm run build && cd dist && nodemon app.js ",
```

Comment:
The purpose of `nodemon` package is to restart your Nodejs app each time you modify something on it. In your case it works the same as node. It was expected, that nodemon will run your `app.js` from `src` frolder, and apply babel build on-flight

Reply:
Fixed.